app/components/views/TicketsPage/MyTicketsTab/TicketInfoCard.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import TicketCard from "./TicketCard";
+import ExpandedInfo from "./ExpandedInfo";
+import Balance from "Balance";
+import { Tooltip } from "shared";
+import { tsToDate } from "helpers/dateFormat";
+import { FormattedMessage as T } from "react-intl";
+
+const statusTxt = {
+  "unknown": <T id="ticket.status.unknown" m="unknown" />,
+  "unmined": <T id="ticket.status.unmined" m="unmined" />,
+  "immature": <T id="ticket.status.immature" m="immature" />,
+  "live": <T id="ticket.status.live" m="live" />,
+  "voted": <T id="ticket.status.voted" m="voted" />,
+  "missed": <T id="ticket.status.missed" m="missed" />,
+  "expired": <T id="ticket.status.expired" m="expired" />,
+  "revoked": <T id="ticket.status.revoked" m="revoked" />,
+};
+
+const TicketInfoCard = ({ ticket, onClick, expanded }) => {
+
+  const className = "ticket-info-card" + (expanded ? " is-expanded" : "");
+  const rewardTipText = <T id="ticket.rewardCalc"
+    m={`Investment: {investment}
+    Transaction Fee: {txFee}
+    Pool Fee: {poolFee}
+    Ticket Reward: {reward}
+    ROI: {roi, number, precise-percent}`}
```

Comment:
Maybe this should make more clear that the ROI is per-ticket? Or no need for that?

Reply:
I think it's fine for now, we'll be adding everything up for clear statistics on the next pass. So for now it's ok to display ROI on a ticket by ticket basis.